src/ui.form-input-search/UInputSearch.vue

```diff
@@ -151,10 +146,14 @@ const { config, searchInputAttrs, searchIconAttrs, clearIconAttrs, searchButtonA
         @click="onClickClear"
       />
 
-      <!-- @slot Use it to add icon after the text. -->
+      <!-- 
+        @slot Use it to add something after the text. 
+        @binding {string} icon-name
+        @binding {string} icon-size      
```

Comment:
`@binding {string} icon-size` but there is no `icon-size` slot prop. remove it at all.